src/sidebar.js

```diff
@@ -200,9 +197,9 @@ class Sidebar extends React.Component {
   }
 
   render() {
-    let sidebarStyle = styles.sidebar,
+    let sidebarStyle = Object.assign({}, styles.sidebar, { zIndex: this.props.zIndex + 1 }),
```

Comment:
Please use `update(orig, {$merge: {…merge objects…}}` syntax. Object.assign is not available in all browsers.

I'm down to polyfill `Object.assign` but only if all calls to `update()` get replaced. I don't want to mix two approaches.